### ISSUE TITLE
Endorsement reminder fails on hidden mods

### ISSUE THREAD

**Issue by Verayth** [Nexus-Mods/web-issues#454]
## Bug/error being reported:
The endorsement reminder window throws an error if you try to endorse a mod that is currently hidden, and there is no way to tell from the list which ones are hidden.

## Steps to reproduce:
1. download mod
2. have author hide mod
3. wait a few days for endorsement reminder window
4. try to endorse the mod
5. hidden mod error message and reminder window goes away

## Expected result:
Either not show the mod in the list at all, or allow hidden mods to be endorsed.

## Actual result:
hidden mod error message. unable to endorse further mods.

**Comment by iamtiz** [comment]
Assigned #154017221

**Comment by iamtiz** [comment]
Hi, which is the mod causing this issue?

**Comment by Verayth** [comment]
I will post the mod/mods as soon as I see them again.  I haven't gotten a reminder window in the last week even though I have reminders set to daily.

**Comment by Verayth** [comment]
I finally received a reminder window again after several months without seeing one.  Clicking on the thumbs up with the first mod in this list "Demoness Bodyslide Preset" throws the hidden mod error message.  There are probably several more hidden mods showing in the list.  Reviewing my download list, the mod is probably number 18426 for fallout 4, which I downloaded on 28 Sep 2016.  I tried endorsing it from the download list, and it also throws the hidden mod error.  So how do I get hidden mods to stop showing up in the reminders window?

![reminders-clipped](https://user-images.githubusercontent.com/8931332/36822231-eeaacd5a-1cc4-11e8-8178-d2d8d07b87c6.png)

![reminders-error-clipped](https://user-images.githubusercontent.com/8931332/36822236-f9588a08-1cc4-11e8-8a62-20159c7cd2d9.png)
